doc/control-mechanisms/reporting.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Reporting Konzept"
+---
+
+## Ziel
+
+Das Ziel dieses Dokuments ist es, die Konzepte zum **Reporting** zentral zu sammeln.
+**Reporting** bezeichnet den Prozess des Erfassens, Bewertens und Speicherns einer Antwort auf
+eine Frage im Dungeon-Kontext.
+
+## Beteiligte Komponenten
+
+**Task-Definition**
+
+Die Task-Definition definiert die Aufgabe, Beispiel:
+
+```
+single_choice_task my_task {
+  answers: ["a", "b", "c"],  // die Antwortmöglichkeiten
+  correct_answer: answers.0, // die korrekte Antwortmöglichkeit
```

Comment:
`answers.0` finde ich umständlich. reicht nicht der index?

Reply:
also `answers[0]`?

Reply:
Oder nur `[0]`?

Reply:
Ich habe die Möglichkeiten dafür hier festgehalten: https://github.com/Programmiermethoden/Dungeon/issues/686

Reply:
ich hab dir in #686 was aus meiner sicht reingeschrieben